packages/intents-sdk/a.ts

```
@@ -0,0 +1,47 @@
+		poaBridgeBaseURL: "",
+		solverRelayBaseURL: "",
+		managerConsoleBaseURL: "",
+		nearIntentsBaseURL: "",
+	},
+});
+
+const now = Date.now();
+const deadline = new Date(now + 1000 * 60 * 5);
+
+console.log(
+	Buffer.from(await sdk["saltManager"].getCachedSalt()).toString("hex"),
```

Biome (2.3.13)

[error] Do not use the global variable Buffer.

Use Uint8Array instead of Node.js Buffer for browser compatibility

(lint/style/noRestrictedGlobals)